### DISCUSSION TITLE
GitHub Copilot could not connect to server.

### DISCUSSION THREAD

**Discussion by Jikuai** [community/community#42290] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

```
vscode version 1.74.2
os: Mac Ventura 13.1
copilot  :v1.65.7705

```

Hi everyone , i got this problem has been four days.

even i re-install VsCode the problem is still, any idea?
`
[ERROR] [default] [2022-12-21T11:46:53.832Z] GitHub Copilot could not connect to server. Extension activation failed: "Connection got disposed."`

**Comment by iamcjwilliams** [comment]
Sorry for the trouble here, please visit this doc to troubleshoot your issue: https://docs.github.com/en/copilot/troubleshooting-github-copilot/troubleshooting-github-copilot-in-visual-studio-code

**Reply by dhust** [reply]
> Comment options
This link doesn't provide troubleshooting help. All it does is tell you how to view the issue you're having. During my programing session I started getting the same error. Thoughts on what it could be?

**Comment by thecloudgeek** [comment]
I'm getting this same error as well... followed the troubleshoot guide and here is what the diagnostics come up with...

```
## Configuration

- HTTP proxy: 
- HTTP proxy autentication: n/a
- Proxy Strict SSL: true
- Extension HTTP proxy support: override

## Environment

- http_proxy: n/a
- https_proxy: n/a
- no_proxy: n/a
- SSL_CERT_FILE: n/a
- SSL_CERT_DIR: n/a
- OPENSSL_CONF: n/a

## Extensions

- Copilot Version: 1.65.7705
- Is `win-ca` installed?: true
- Is `mac-ca` installed?: true

## Node setup

- Number of root certificates: 131
- Operating system: Darwin
- Operating system version: 22.1.0
- Operating system architecture: arm64
- NODE_OPTIONS: n/a
- NODE_EXTRA_CA_CERTS: n/a
- NODE_TLS_REJECT_UNAUTHORIZED: n/a
- tls default min version: TLSv1.2
- tls default max version: TLSv1.3

## Reachability

- github.com: HTTP 200 - 
- copilot-proxy.githubusercontent.com: HTTP 200 - 
- default.exp-tas.com: HTTP 200 - 
```

interesting thing is that it works in Intellij for me but not in VSCode...
